common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java

```diff
@@ -218,6 +226,11 @@ public static String getPropagator() {
     return getProperty(OTEL_PROPAGATORS, DEFAULT_PROPAGATOR);
   }
 
+  public static boolean useInsecureTransport() {
```

Comment:
Can we make this method private? I don't think this needs to be public (if **yes**, then let's mark this as deprecated).

Reply:
no, it's used in launcher (manual configuration)